pytx3/pytx3/signal_type/raw_text.py

```diff
@@ -24,5 +29,69 @@ class RawTextSignal(signal_base.SimpleSignalType, signal_base.StrMatcher):
     INDICATOR_TYPE = "DEBUG_STRING"
     TYPE_TAG = "media_type_text"
 
+    def __init__(self) -> None:
+        super().__init__()
+        self.normal_to_raw = {}
+
     def match(self, content: str) -> t.List[signal_base.SignalMatch]:
         return self.match_hash(content)
+
+    def match_hash(self, signal_str: str) -> t.List[signal_base.SignalMatch]:
+        normalized_str = self._normalize_string(signal_str)
+        # Match considered if 95% match
```

Comment:
Should we make 95% a constant?

Reply:
Eh, I prefer not to constant-tize if only used in one place in the code.

In the long term, I think we'll likely made this configurable in some way, but I'm not sure how yet (I originally had an entry in the collab config for it).